specs/schemes/exact/scheme_exact_algo.md

```diff
@@ -0,0 +1,173 @@
+# Scheme: `exact` on `Algorand`
+
+## Summary
+
+The `exact` scheme on Algorand uses the `Algorand Standard Asset (ASA)`, which are native assets (no contract required) on the Algorand blockchain to authorize a transfer of a specific amount from the payer to the resource server. The approach results in the facilitator having no ability to direct funds anywhere but the address specified by the resource server in `paymentRequirements`.
+
+## Payment Requirements asset definition 
+
+In the `exact` scheme on Algorand, the `paymentRequirements.asset` must be a string representing an `Algorand Standard Asset (ASA)` ID unsigned integer instead of an `ERC20` contract address. This **MUST** be validated by the resource server to ensure the `asset` field is valid when using the Algorand scheme.
+
+
+## `X-Payment` header payload
+
+The `payload` field of the `X-PAYMENT` header must contain `paymentGroup` as a field. It represents an atomic group of transactions as an array. This is natively supported by Algorand and does not require contracts.
+
+A group can have transactions of different types, such as `pay` (payment) and `axfer` (asset transfer). 
+
+There can be multiple signers in the group and fees can be pooled together or assigned to a specific signer; meaning it can be delegated to a specific account to pay the fees for the group. A max of 16 transactions can be included in a group.
+
+Example:
+
+```json
+{
+  "paymentGroup": [
+    {
+      "signature": "wVGxbZwPd2c8zKX4RSE4muFRBOHe/nRq5VT6iMeMksi2agfOCTs+VzOGUPJ+bTT5T21EuYjoBKKDX2+/NZG8Cg==",
+      "txn": {
+        "gen": "mainnet-v1.0",
+        "gh": "SGO1GKSzyE7IEPItTxCByw9x8FmnrCDexi9/cOUJOiI=",
+        "asnd": "PPNHVQJGE6ZMEWPR35UHLUYMCCZV6VNTHLJMZDVCONXKUPV47K4WGQJWGI",
+        "arcv": "ACTSMNPJPS5JMZJJ5G73JOXUUMWXXDGS7TMOER3M4W7BC54ERSZ4KPSMHM",
+        "type": "axfer",
+        "fee": "0",
+        "fv": "6631154",
+        "lv": "6632154",
+        "xaid": "168103"
+      }
+    },
+    {
+      "signature": "6iYMGg8cMw0VYYCfmwru8lixfrVyi1LFXm9H/M0MGf0P4vjK6KGsiCUvsocWUvMo8yTMWQ0hFcP86sRoyGlIAg==",
+      "txn": {
+        "gen": "mainnet-v1.0",
+        "gh": "SGO1GKSzyE7IEPItTxCByw9x8FmnrCDexi9/cOUJOiI=",
+        "fee": "2000",
+        "fv": "6631154",
+        "lv": "6632154",
+        "snd": "LOXIQKHRCEDEMN5MKBQ32Y54J7H6JKBTEURQL4S65K44MTWN6UMTU2WNBM",
+        "rcv": "LOXIQKHRCEDEMN5MKBQ32Y54J7H6JKBTEURQL4S65K44MTWN6UMTU2WNBM",
+        "amnt": "0",
+        "type": "pay"
+      }
+    }
+  ]
+}
+```
```

Comment:
I think the group (`grp`) field is missing in this example. Also, what if we have a single transaction? Should we skip the group (`grp`) field in that case?

Reply:
Right. good catch. will add